website/pages/api-docs/system/lease-count-quotas.mdx

```diff
@@ -0,0 +1,102 @@
+---
+layout: api
+page_title: /sys/quotas/lease-count - HTTP API
+sidebar_title: <code>/sys/quotas/lease-count</code>
+description: The `/sys/quotas/lease-count` endpoint is used to create, edit and delete lease count quotas.
+---
+
+# `/sys/quotas/lease-count`
+
+~> **Enterprise Only** – These endpoints require Vault Enterprise.
+
+The `/sys/quotas/lease-count` endpoint is used to create, edit and delete lease count quotas.
+
+## Create or Update a Lease Count Quota
+
+This endpoint is used to create a lease count quota with an identifier, `name`.
+A lease count quota must include a `max_leases` value with an optional `path`
+that can either be a namespace or mount.
+
+| Method | Path                            |
+| :----- | :------------------------------ |
+| `POST` | `/sys/quotas/lease-count/:name` |
+
+### Parameters
+
+- `name` `(string: "")` - The name of quota.
+- `path` `(string: "")` - Path of the mount or namespace to apply the quota.
+  A blank path configures a global lease count quota. For example `namespace1/`
+  adds a quota to a full namespace, `namespace1/auth/userpass` adds a quota to
+  `userpass` in `namespace1`. **Note, namespaces are supported in Enterprise only**.
```

Comment:
The note about namespaces could be confusing IMO since the entire endpoint set was explained to be Enterprise only.

Reply:
Thanks @kalafut -- this was taken from the expert from the rate limit quota doc. I'll remove this.